crrExistingObjects.js

```diff
@@ -95,7 +95,11 @@ function _markObjectPending(bucket, key, versionId, storageClass,
             VersionId: versionId,
         }, log, next),
         (mdRes, next) => {
+            if (mdRes === undefined) {
+                return next(new Error('could not get metadata'));
```

Comment:
If this is an error, it could be more appropriate to raise it from `getMetadata` (better details about the error, more consistent).